presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -529,6 +531,8 @@ public Lock getExclusiveLock()
             System.out.println(PlanPrinter.textLogicalPlan(plan.getRoot(), plan.getTypes(), metadata, session));
         }
 
+        planVerifiersFactory.verify(plan);
```

Comment:
I don't understand the question. There is no such SQL syntax as `verify`. Here verfiy is called at the end of  planning.